### DISCUSSION TITLE
Question: style doesn't show correctly with WinUI 2 on my machine.

### DISCUSSION THREAD

**Discussion by cdytoby** [microsoft/microsoft-ui-xaml#7429] [Q&A]
I'm not sure if this is correct place to ask, I asked one on reddit but no response.

Windows 10, version 21H1

I don't know when or why it happens, it happens sometime after I updated to 21H1 since it says I'm available to upgrade, also after sometime Microsoft Store is updated to newest version with new UI. Then I noticed something is totally wrong with the UI in Microsoft Store, here are the screenshots:

https://imgur.com/a/YHoRzjH

Not just Microsoft Store, I tried multiple things, like Feedback Center, Powertoys, all have same issue. Then through powertoys I realized they're using WinUI 2 framework, so I downloaded the XAML Controls Gallery, as expected, all of them have the same issue. Screenshots already in the above link.

Toggle and others are not so annoying, but the buttons are very very annoying, because unless I press the button, I can't read the text.

What exactly happened and what should I do to fix it?

Also, I sent a feedback in feedback center, but there are no response at all.
https://aka.ms/AAfdy5b

**Comment by StephenLPeters** [comment]
Duplicate of https://github.com/microsoft/microsoft-ui-xaml/issues/6344 and https://github.com/microsoft/PowerToys/issues/14743  the issue appears to be hardware related. we are still investigating.
